docs/source/conf.py

```diff
@@ -372,6 +372,10 @@
     "css_files": ["_static/theme_overrides.css"]  # override wide tables in RTD theme
 }
 
+linkcheck_ignore = [
```

Comment:
Is this something you used to find broken links or does this change sphinx configuration somehow ?

Reply:
It's only used for the link-checking and is mostly there to avoid hitting rate limits when checking github links (mostly in the changelog)

Reply:
Thanks!  Also came upon https://github.com/dask/dask/issues/4058#issuecomment-657082200 for additional context

Reply:
Excellent, apologies if I made that a bit convoluted! :-)